cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsTerminateService.java

```diff
@@ -97,8 +97,8 @@ private void waitAndDeleteCloudformationStack(AuthenticatedContext ac, CloudStac
         DescribeStacksRequest describeStacksRequest = new DescribeStacksRequest().withStackName(cFStackName);
         try {
             retryService.testWith2SecDelayMax15Times(() -> isStackExist(amazonCloudFormationClient, cFStackName, describeStacksRequest));
-        } catch (ActionFailedException ignored) {
-            LOGGER.debug("Stack not found with name: {}", cFStackName);
+        } catch (ActionFailedException | AmazonServiceException e) {
```

Comment:
Can we have a separate catch for this particular case, that clearly states we did not found the stack?
Isn't it possible that for some reason we get ActionFailed, like "expired credential", the we proceed with the delete?

Reply:
Yes, you are right.